File: app/src/main/java/id/ac/ui/cs/mobileprogramming/nabilafv/rememberyourlord/api/ApiHelperInterface.java
package id.ac.ui.cs.mobileprogramming.nabilafv.rememberyourlord.api;

import androidx.lifecycle.LiveData;

import id.ac.ui.cs.mobileprogramming.nabilafv.rememberyourlord.model.WeatherResponse;
import retrofit2.Response;

public interface ApiHelperInterface {
    LiveData<Response<WeatherResponse>> getWeather(double lat, double lan);
}

File: app/src/main/java/id/ac/ui/cs/mobileprogramming/nabilafv/rememberyourlord/api/ApiHelperInterface.kt
package id.ac.ui.cs.mobileprogramming.nabilafv.rememberyourlord.api

import id.ac.ui.cs.mobileprogramming.nabilafv.rememberyourlord.model.WeatherResponse
import retrofit2.Response

interface ApiHelperInterface {
    suspend fun getWeather(lat: Double, lon: Double): Response<WeatherResponse>
}